src/Fixer/Whitespace/MethodChainingIndentationFixer.php

```diff
@@ -59,30 +59,37 @@ protected function applyFix(\SplFileInfo $file, Tokens $tokens): void
                 continue;
             }
 
+            $endParenthesisIndex = $tokens->getNextTokenOfKind($index, ['(', ';', ',', [T_CLOSE_TAG]]);
+
+            if (null === $endParenthesisIndex || !$tokens[$endParenthesisIndex]->equals('(')) {
```

Comment:
Does that mean properties are not indented in a case such as e.g.:
```php
$a = $b->
   ->c()
  ->d;
```

Reply:
I assume so, however the name of this rule implies it will only touch chained method calls. Surely this leaves the question as why not fix class attributes as well in a similar way.
Might be worth looking into and creating a new rule or updating this one etc!

Reply:
Yes, I would update the rule. I just noticed #6577 which shows this can be the expected behavior 😄.

Reply:
oh no :fearful: I just saw the response on 6577 ... hope to find some time squashing that bug as well but might be a few days cause of other timings and such